File: Coding Interview Prep/multiples_of_3_and_5.py
# sum of multiples of 3 and 5 for FCC practice. Pretty straightforward.

def multiples_of_3_and_5(number):
    total = 0
    for value in range(0, number):
        if value % 3 == 0 or value % 5 == 0:
            total += value
    return total

File: FCC_Coding_Interview_Prep/multiples_of_3_and_5.js
// sum of multiples of 3 and 5.
// Javascrip doesn't seem to have a range() function like Python does
// but this can be replaced with an incrementing for loop.
function multiplesOf3and5(number) {
    let sum = 0;
    for(let i = 0; i < number; i++) {
      if (i % 3 == 0 || i % 5 == 0) {
        sum += i;
      }
    }
    return sum;
}
    
